src/IO/WriteBufferFromBlobStorage.cpp

```diff
@@ -0,0 +1,73 @@
+#if !defined(ARCADIA_BUILD)
+#include <Common/config.h>
+#endif
+
+#if USE_AZURE_BLOB_STORAGE
+
+#include <IO/WriteBufferFromBlobStorage.h>
+
+
+namespace DB
+{
+
+// TODO: abstract this function from DiskS3.cpp, from where it was copy-pasted
```

Comment:
If you  want to remove copy-paste then it is possible to move it to some `RemoteDisksCommon` file.